Modules/CodeTrek/Services/CodeTrekService.php

```diff
@@ -109,4 +110,17 @@ public function moveApplicantToRound($applicant, $data)
         $applicationRound->start_date = $data['start_date'];
         $applicationRound->save();
     }
+
+    public function storeCodeTrekApplicantFeedback($data)
+    {
+        $feedback = new CodeTrekCandidateFeedback();
+        $feedback->category_id = $data['feedback_category'];
+        $feedback->feedback = $data['feedback'];
+        $feedback->feedback_type = $data['feedback_type'];
+        $feedback->latest_round_name = $data['latest_round_name'];
+        $feedback->candidate_id = $data['candidate_id'];
+        $feedback->posted_by = auth()->user()->id;
+        $feedback->posted_on = date('y-m-d');
```

Comment:
This carbon function will also work
```suggestion
        $feedback->posted_on = today();
```